src/ripple/protocol/TER.h

```diff
@@ -135,6 +135,8 @@ enum TEMcodes : TERUnderlyingType {
     temXCHAIN_BRIDGE_BAD_REWARD_AMOUNT,
 
     temEMPTY_DID,
+
+    temARRAY_SIZE,
```

Comment:
nit: should be `temBAD_ARRAY_SIZE` or `temARRAY_TOO_LONG` or something like that